pyprophet/io/_base.py

```diff
@@ -619,6 +619,61 @@ def export_results(self, data: pd.DataFrame):
         else:
             raise ValueError(f"Unsupported export format: {cfg.export_format}")
 
+    def clean_and_export_library(self, data: pd.DataFrame) -> pd.DataFrame:
+        """
+        This function cleans the original dataframe and exports the library
+
+        Args:
+            data: Input DataFrame with library data
+
+        """
+        cfg = self.config
+
+        # For precursors found in more than one run, select the run with the smallest q value
+        # If q values are the same, select the first run
```

Comment:
Would it make a difference if we sort by intensity as well? So if there are ties in peakgroup q-values, then we select the highest intensity transition, and lastly if there are ties in both q-values and intensity, then select the first entry. Or would it not make too much of a difference?